### ISSUE TITLE
Recycle Issues

### ISSUE THREAD

**Issue by ElderSuccubus** [abearcodes/Valheim#64]
Sometimes when recycling an item it will instead recycle a different item. Sometimes when even selecting the item alone it will deselect or select something different in a matter of moments to a couple seconds later.

**Comment by TwentyOneZ** [comment]
Yeah, confirmed here. You can click to recycle item#1, then select item#2 before the crafting bar is done. This will recycle item#2, even if  you don't have the requirements for doing so.
